src/Fixture/AbstractAssetFixture.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\Fixture;
+
+use Pimcore\Model\Asset;
+use Pimcore\Model\Asset\Image;
+
+abstract class AbstractAssetFixture extends AbstractFixture
+{
+    public function __construct(
+        protected string $fullQualifiedFilename,
+        protected string $pimcoreFilename,
+        protected string $pimcoreBasePath,
+        protected string $fixtureAssetMarker,
+    ) {
+    }
+
+    public function create(): void
+    {
+        $asset = new Image();
+        $asset->setFilename($this->pimcoreFilename);
+        $asset->setData(file_get_contents($this->fullQualifiedFilename));
```

Comment:
I'm not sure if it is a good idea directly implementing the `file_get_contents` mechanism inside this Fixture base class.
We ran in other environments into trouble loading file content by HTTP requests which would also be possible by calling this method.

I would change the signature and set first parameter to `protected string $fileContent`.